### DISCUSSION TITLE
Results returning SQL Statement

### DISCUSSION THREAD

**Discussion by Datatatata** [dbeaver/dbeaver#22921] [Q&A] [answered]
### Description

Hi,
I have a problem since recently that when executing a statement the result tab appear showing the SQL statement I executed instead of extracting data. Would anyone know how to fix this?
![Dbeaver bug](https://github.com/dbeaver/dbeaver/assets/160870529/0a98d00c-de52-4f57-8310-efc440ab30b8)

### DBeaver Version

21.0.1

### Operating System

Windows 64-bit operating system, x64-based processor

### Database and driver

Vertica ADB

**Comment by E1izabeth** [comment, marked as answer]
Could you try clicking on this arrow?
![image](https://github.com/dbeaver/dbeaver/assets/28875055/65ab6e97-90dc-4219-8604-079a57d1d738)

**Comment by E1izabeth** [comment]
I suppose you clicked the Collapse arrow accidentally some time ago, that's why the left-side area was collapsed.
The issue is solved now, right?

**Comment by Datatatata** [comment]
Yes I think it is fine now, thanks Elizabeth =) !
